src/Moryx/Moryx.csproj

```diff
@@ -6,6 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Description>Core package of the MORYX ecosystem. It defines the necessary types for MORYX compatibility as well as commonly required tools.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>json;modularity;IoT;IIoT;industrial;IoT framework;newtonsoft;foundation</PackageTags>
```

Comment:
do not use newtonsoft here, instead e.g. our core namespaces

e.g.: "MORYX;Serialization;Configuration; ......."